src/main/java/com/odysseusinc/arachne/portal/repository/UserSpecifications.java

```diff
@@ -48,6 +50,17 @@
         return (root, query, cb) -> cb.like(root.get(field), namePattern);
     }
 
+    public static <U extends IUser> Specification<U> usersIn(final Long[] tenantIds) {
+
+        return ((root, query, cb) -> {
+            final Path<Tenant> tenantIdPath = root.join("tenants").get("id");
+            query.groupBy(root.get("id"));
+            query.having(cb.gt(cb.count(root), tenantIds.length - 1));
```

Comment:
Could you explain me how length of identities array related with number of users within tenant?